Guardfile.example

```diff
@@ -0,0 +1,73 @@
+# More info at https://github.com/guard/guard#readme
+
+# You'll need to make sure Guard, and any of its plugins are in your Gemfile.local
+#
+# Example:
+#  # Automatically run tests on file changes
+#  gem 'guard', require: false
+#  gem 'guard-rspec', require: false
+#  gem 'guard-bundler', require: false
+#  gem 'terminal-notifier-guard', require: false
+
```

Comment:
Maybe mention that after bundle installing/updating, guard should be run via `bundle exec guard`?

Reply:
Done.